final_prep/README.md

```diff
@@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1055" alt="Screen Shot 2021-09-11 at 2 38 24 PM" src="https://user-images.githubusercontent.com/39220746/132961125-da3c5444-01dd-4522-b849-b2e6eb00ede9.png">
+
+- <img width="1051" alt="Screen Shot 2021-09-11 at 2 38 12 PM" src="https://user-images.githubusercontent.com/39220746/132961150-4707ba46-e5d7-4b0b-b468-15708943245c.png">
+
+- <img width="1063" alt="Screen Shot 2021-09-11 at 2 37 58 PM" src="https://user-images.githubusercontent.com/39220746/132961156-bb014f93-2e8d-4734-a20a-aa3000d27494.png">
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: <script src="https://gist.github.com/CooleyNick/25143b8d481cd3cd7a215b437e5942e7.js"></script>
```

Comment:
I was able to access these links, because the url is visible, but check how your markdown ended up [here](https://github.com/CooleyNick/backend_mod_1_prework/tree/main/final_prep) - review the syntax for links when using md. 

Then try going to those urls. I see code like blurbs. Check out your file extensions- right now, these are javascript files.

Make sure to fix these links and files for completion of prework.